docs/setup-cluster/deploy-cluster/firewall-rules.rst

```diff
@@ -22,6 +25,8 @@ The firewall rules must satisfy the following network access requirements for th
  Agents
 ********
 
+NEEDS UPDATES
```

Comment:
The only thing that I think that might be missing is for generic proxies.

Do you know if we have to update any port requirements for that @ioga ?